_rules/auto-update-text-efbfc7.md

```diff
@@ -20,18 +20,18 @@ acknowledgements:
 
 ## Applicability
 
-The rule applies to any [visible][] [text content][] in an [HTML document][] if:
+The rule applies to any [visible][] [HTML element][] in an [HTML document][] if:
 
-- the [text content][] is removed or replaced by new content that does not contain the previous content; and
-- the removal or replacing is not caused by a [user interaction][]; and
-- the removal or replacing happens anytime after the [readiness][document readiness] of the [HTML document][] the [text node][] belongs to is equal to "complete"; and
+- the `innerText` property of the [element][html element] is updated with a determinable frequency;
```

Comment:
I think "updated" is not precise enough. For example, it is not clear whether "deleting" is "updating". Or whether overwriting with the same value is "updating". Maybe going in the direction of "changed" is more precise 🤔.

Reply:
Agreed and reverted back to "changes" instead of "updates".